### ISSUE TITLE
Merge the welcome and about pages

### ISSUE THREAD

**Issue by AlbertoPdRF** [AlbertoPdRF/tisn.app#304]
**Is your feature request related to a problem? Please describe.**

The welcome page is quite ugly and empty at the moment, and it's main call to action drives users to the about page. The about page will hopefully be way more visually appealing after #303, and it already has a call to action to drive users to register, so merging both will hopefully improve our user conversion rates! 

**Describe the solution you'd like**

I'd like the welcome and about pages to be merged into one, but also keep serving the `About` component on the `/about` route for logged in users to be able to see it.

**Describe alternatives you've considered**

Leave everything as is.

**Additional context**

None.

**Comment by ch374n** [comment]
hey, I would like to work on this one. can you please give some detailed information ? thank you.

**Comment by AlbertoPdRF** [comment]
Hi @ch374n, welcome to our repository and thanks for your interest on this issue! :raised_hands:

Here's a more detailed explanation of what this issue is about: right now we have a `Welcome` component that gets rendered as the landing page for non-signed up/logged in users. We also have an `About` component that gets rendered whenever anyone visits the `/about` route. The idea is to render the `About` component inside the `Welcome` one -- making the necessary adjustments so we end up having a nice landing page --, but also keep rendering the standalone `About` component when logged in users visit the `/about` route.

I hope that's a bit clearer! :slightly_smiling_face: But if you need anything else, don't hesitate to ask! And you can also join our Slack workspace, for which you can find an invite link on the README.

If you want to work on the issue, just let me know and I'll assign it to you! :eyes:

**Comment by ch374n** [comment]
Okay then assign me

**Comment by AlbertoPdRF** [comment]
Done! :smiley:
